DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -126,6 +126,23 @@ private function sessionLoad($config, ContainerBuilder $container)
                 case 'jackrabbit':
                     if (empty($loaded['jackalope'])) {
                         $this->loader->load('jackalope.xml');
+
+                        // TODO: move the following code block back into the XML file when we drop support for symfony <2.6
+                        $jackalopeTransports = array('prismic', 'doctrinedbal', 'jackrabbit');
+                        foreach ($jackalopeTransports as $transport) {
+                            $factoryServiceId = sprintf('doctrine_phpcr.jackalope.repository.factory.service.%s', $transport);
+                            $factoryService = $container->getDefinition(sprintf('doctrine_phpcr.jackalope.repository.factory.%s', $transport));
```

Comment:
thanks for the input. $factoryDefinition seems the right name. i updated
the code.